example_client/benchmark/entrypoint.sh

```diff
@@ -0,0 +1,144 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#! /bin/bash
+echo "OVMS benchmark client 1.11"
```

Comment:
Do we want to keep separate versioning of perf client? If so will we somewhat provide information with which OVMS version it is supposed to work?

Reply:
I would like to keep separated version because we use and develop this client in validation

Reply:
we can also change to 2022.1.11